src/masoniteorm/schema/platforms/SQLitePlatform.py

```diff
@@ -0,0 +1,143 @@
+
+
+class SQLitePlatform:
+
+    type_map = {
+        "string": "VARCHAR",
+        "char": "CHAR",
+        "integer": "INTEGER",
+        "big_integer": "BIGINT",
+        "tiny_integer": "TINYINT",
+        "big_increments": "BIGINT",
+        "small_integer": "SMALLINT",
+        "medium_integer": "MEDIUMINT",
+        "increments": "INTEGER PRIMARY KEY",
+        "binary": "LONGBLOB",
+        "boolean": "BOOLEAN",
+        "decimal": "DECIMAL",
+        "double": "DOUBLE",
+        "enum": "ENUM",
+        "text": "TEXT",
+        "float": "FLOAT",
+        "geometry": "GEOMETRY",
+        "json": "JSON",
+        "jsonb": "LONGBLOB",
+        "long_text": "LONGTEXT",
+        "point": "POINT",
+        "time": "TIME",
+        "timestamp": "TIMESTAMP",
+        "date": "DATE",
+        "year": "YEAR",
+        "datetime": "DATETIME",
+        "tiny_increments": "TINYINT AUTO_INCREMENT",
+        "unsigned": "INT UNSIGNED",
+        "unsigned_integer": "UNSIGNED INT",
+    }
+
+    def compile_create_sql(self, table):
+        columns = self.columnize(table.get_added_columns())
+        
+        return self.create_format().format(
+            table=self.table_string().format(table=table.name).strip(),
+            columns=', '.join(self.columnize(table.get_added_columns())).strip(),
+            constraints=', ' + ', '.join(self.constraintize(table.get_added_constraints())) if table.get_added_constraints() else ""
+        )
+
+    def compile_alter_sql(self, diff):
+        sql = []
+
+        if diff.removed_indexes:
+            for name, index in diff.removed_indexes.items():
+                sql.append("DROP INDEX {name}".format(name=index.name))
+        
+        if diff.added_columns:
+            for name, column in diff.added_columns.items():
+                sql.append("ALTER TABLE {table} ADD COLUMN {name} {data_type}".format(
+                    table=diff.name, 
+                    name=column.name,
+                    data_type=self.type_map.get(column.column_type, ""),
+                ).strip())
+        
+        if diff.renamed_columns:
+            sql.append("CREATE TEMPORARY TABLE __temp__{table} AS SELECT {original_column_names} FROM {table}".format(
+                table=diff.name,
+                original_column_names=', '.join(diff.from_table.added_columns.keys())
+            ))
+            sql.append("DROP TABLE {table}".format(table=diff.name))
+
+            columns = diff.from_table.added_columns
+            columns.update(diff.renamed_columns)
```

Comment:
with the table and table diff structure we can do something cool like this where we have the columns added on the table and then we can overwrite them with the renamed columns. Nice little short piece of effective code.